src/data/markdown/translated-guides/en/02 Using k6/02 Metrics/100 Create custom metrics.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Create custom metrics
+description: How to build custom k6 metrics for each metric type.
+---
+
+Besides the [built-in metrics](/results-output/metrics-reference), you can create custom metrics.
+For example, you can create some derived metrics from the built-in metrics, or use the [Response.timings](/javascript-api/k6-http/response) object to create a metric for a specific set of endpoints.
+
+Each metric type has a [constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor) to create a custom metric.
+The constructor creates a metric object of the declared type. Each type has methods to take metric measurements.
+
+If you need help constructing a custom metric, read the following sections of this document.
+They document the procedure and provide examples.
+
+If you're comfortable with these Javascript conventions already, you might prefer to read the [reference documentation for each metric constructor](/javascript-api/k6-metrics).
```

Comment:
```suggestion
If you're comfortable with these already, you might prefer to read the [reference documentation for each metric constructor](/javascript-api/k6-metrics).
```
I'm not getting why they are JavaScript _conventions_

Reply:
I think I wanted to avoid saying "methods" :-)